website/views.py

```diff
@@ -2047,6 +2047,19 @@ def update_comment(request, issue_pk, comment_pk):
     }
     return render(request, "comments2.html",context)
     
+def delete_comment(request):
+    int_issue_pk = int(request.POST['issue_pk'])
+    issue = Issue.objects.get(pk=int_issue_pk)
+    if request.method == "POST":
+        all_comment = Comment.objects.filter(issue=issue)
```

Comment:
All comment is loaded below so you can remove this too

Reply:
"all_comment" for sir fetch whole comment is web page after delete the comment